### DISCUSSION TITLE
Disable system user in xrdp

### DISCUSSION THREAD

**Discussion by amicoderozer** [neutrinolabs/xrdp#1953] [Q&A] [answered]
Good Morning everyone,
I am using xrdp to remote access my linux server. It works great and I love it! 
I was wondering if there is a method to configure xrdp to only allow remote login some users.
I explain myself, I have two users in my system. The first one is the one I use when I use the server like a normal pc (is connected to my tv), so it has a normal not really strong password and the second user is the one I use to connect throw ssh. And this one has a bullet proof password. In my ssh config I disabled the first one, so only the user with the strong password is allowed to connet remotely. 
Is there by any change a similar feature in xrdp?

Thank you very much

**Comment by metalefty** [comment, marked as answer]
I think `TerminalServerUsers` in `sesman.ini` can be used for such a purpose.

Create a group like `xrdpusers` and set `TerminalServerUsers=xrdpusers`, invite the users you want to allow login with xrdp to that group.

**Reply by amicoderozer** [reply]
Ok thank you for the fast reply, I'll try and let you know!

**Reply by amicoderozer** [reply]
It worked, thank you very much!!
